src/components/MenuItem.js

```diff
@@ -86,6 +86,7 @@ const MenuItem = (props) => {
                 styles.popoverMenuItem,
                 StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive), true),
                 ..._.isArray(props.wrapperStyle) ? props.wrapperStyle : [props.wrapperStyle],
+                styles.popoverMaxWidth,
```

Comment:
Out of curiosity, why add this style here instead of to `descriptionTextStyle` like you suggested in the proposal?

Reply:
That makes sense, and I agree this is cleaner, thanks!